python/hail/representation/pedigree.py

```diff
@@ -0,0 +1,305 @@
+from hail.java import *
+
+
+class Trio(object):
+    """Class containing information about nuclear family relatedness and sex.
+
+    :param str proband: sample ID of proband
+
+    :param father: sample ID of proband
+    :type father: str or None
+
+    :param mother: sample ID of proband
+    :type mother: str or None
+
+    :param sex: sex of proband: 1 for male, 2 for female
+    :type sex: int or None
+    """
+
+    _sex_jobject = None
+    _pheno_jobject = None
+
+    @staticmethod
+    def _get_sex_jobject():
```

Comment:
You don't need to cache these.  As far as I can tell, they can only be used once.